x-pack/plugins/infra/public/pages/metrics/hosts/components/kpis/kpi.tsx

```diff
@@ -64,11 +64,11 @@ export const Kpi = ({ id, title, layers, toolTip, height }: KPIChartProps & { he
       dateRange={afterLoadedState.dateRange}
       filters={afterLoadedState.filters}
       layers={layers}
-      lastReloadRequestTime={afterLoadedState.lastReloadRequestTime}
       loading={loading}
       height={height}
       query={afterLoadedState.query}
       title={title}
+      searchSessionId={afterLoadedState.searchSessionId}
```

Comment:
"We can initiate new searches by updating the searchSessionId. This allows us to replace the lastReloadRequestTime prop, whose sole purpose was to ensure that."